camshare/camshare/ViewControllers/PhotoAlbumViewController.swift

```diff
@@ -121,13 +121,23 @@ class PhotoAlbumViewController: ViewController, AVCaptureMetadataOutputObjectsDe
             self.captureQRCode()
         }
 
+        let signOut = UIAlertAction(title: "Sign Out", style: .destructive) { (_) in
+            do {
+                try Auth.auth().signOut()
+                exit(0)
```

Comment:
I know why you have done sign out this way 😆 . I did the exact same thing for my project and was told that iOS Apps should never terminate randomly (intentionally). 

This is easier but the more correct approach is to actually log out and then redirect the user to the login page.

Reply:
Ahh yes thanks man, meant to take it out